graphql_compiler/compiler/filters.py

```diff
@@ -860,6 +868,10 @@ def _get_filter_op_name_and_values(directive):
     u'is_null',
     u'is_not_null',
 })
+UNARY_FILTERS = frozenset({
```

Comment:
I can't make up my mind on whether this is confusing or not. On one hand, the filter _is_ unary because `is_null` operates on the field. On the other, it's actually a nullary operation because it doesn't take any arguments. @bojanserafimov what do you think?

Reply:
That's a good point, would we traditionally consider something like `has_substring` to be binary?

Reply:
I think unary makes sense. It's even pythonic, with `self` being a function argument in class methods.